src/common/components/__tests__/__snapshots__/CitationItem.test.jsx.snap

```diff
@@ -30,13 +23,13 @@ exports[`CitationItem renders with full citation 1`] = `
           publicationInfo={
             Immutable.List [
               Immutable.Map {
-                journal_title: "Journal",
+                "journal_title": "Journal",
               },
             ]
           }
           wrapperClassName={null}
         />
-      </Unknown>
+      </React.Fragment>
```

Comment:
why in some tests is React.Fragment and some others Fragment? it has to do with how we import things?

Reply:
I haven't checked it yet, but we always `import { Fragment }`

Reply:
I don't think this is a real issue but it seems like it's `Fragment` if it is the root node, and `React.Fragment` if it is passed as prop to another component.